---
layout: page
title: research
permalink: /research/
description: an overview of current and past research projects
nav: true
nav_order: 3
horizontal: false
---

#### **Rice University**

Advised by [Dr. C&eacute;sar A. Uribe](https://cauribe.rice.edu/) and [Dr. Lydia Beaudrot](https://lydiabeaudrot.weebly.com/).

**TL;DR:** A systemic understanding of predator-prey relationships is central to maintaining biodiversity and the well-being of an ecosystem. As anthropogenic disturbance continues to rise in Sub-Saharan Africa, we are in need of a principled approach to study and anticipate changes among its food web networks as a result of human interference. We leverage optimal transport as the backbone of our newly proposed geometric data analysis toolkit over food web networks. Check out our preliminary results below!

[1] **Hung, K.**, Zalles, A., et al. (2024). Towards Ecological Network Analysis with Optimal Transport. ICLR 2024 Workshop on Tackling Climate Change with Machine Learning.

[2] Zalles, A., **Hung, K.**, et al. (2023). [Network Regression with Wasserstein Distances](/assets/papers/zalles2023_network_regression.pdf). NeurIPS 2023 Workshop on Optimal Transport and Machine Learning.

<div class="row mt-2">
    <div class="col-sm-6 mt-3 mt-md-0">
        {% include figure.html path="assets/img/research/food-web-ot.png" title="A Gromov-Wasserstein Approach for Food Web Network Analysis" class="img-fluid rounded z-depth-1" zoomable=true %}
    </div>
    <div class="col-sm-6 mt-3 mt-md-0">
        {% include figure.html path="assets/img/research/network-reg-ot.png" title="Network Regression with Wasserstein Distances" class="img-fluid rounded z-depth-1" zoomable=true %}
    </div>
</div>
<hr/>

#### **New York University**
Advised by [Dr. Esteban G. Tabak](https://cims.nyu.edu/~tabak/). Supported by the [AM-SURE](https://math.nyu.edu/dynamic/research/pages/research-and-training-group-mathematical-modeling-and-simulation/activities/amsure/) program, funded by NSF.

**TL;DR:** Often times, your data is affected by variables that should be excluded from your analysis (for ex. batch effect and protected attributes such as race, gender, socioeconomic class). We developed a theoretical framework for reducing the unwanted variability in our data distribution via optimal transport. Look out for our future works to extend this method for personalization, latent factor discovery, and semi-supervised variability reduction! 

<div class="row mt-2" style="height: 35rem;">
    <div class="col-sm-6 mt-3 mt-md-0">
        <iframe src="/assets/pdf/var-reduce-ot-presentation.pdf" width="100%" height="50%"></iframe>
        <br>
        <p>
            <a href="/assets/pdf/var-reduce-ot-presentation.pdf" target="_blank" rel="noopener noreferrer">
                <!-- You can use an icon or text as a link for the PDF viewer -->
                View Slides (PDF)
            </a>
        </p>
    </div>
    <div class="col-sm-6 mt-3 mt-md-0">
        <iframe src="/assets/pdf/var-reduce-ot-paper.pdf" width="100%" height="95%"></iframe>
        <br>
        <p>
            <a href="/assets/pdf/var-reduce-ot-paper.pdf" target="_blank" rel="noopener noreferrer">
                <!-- You can use an icon or text as a link for the PDF viewer -->
                View Paper (PDF)
            </a>
        </p>
    </div>
</div>

<hr/>

#### **Berkeley AI Research**

Advised by [Dr. Alison Gopnik](http://www.gopniklab.berkeley.edu/alison) and [Eunice Yiu](https://www.linkedin.com/in/euniceyiu). Supported by the [SUPERB](https://eecs.berkeley.edu/resources/undergrads/research/superb) and [BAIR](https://bair.berkeley.edu/reu.html) NSF REU programs. 

**TL;DR:** Have you ever contrasted the intelligence of children with adults, and maybe even artificial intelligence? Children are known to learn quickly and generalize accurately with little data to their disposal, contrary to deep neural networks and other modern machine learning frameworks. Our analysis is two-fold: (1) to understand the discrepancies in children and adults' exploration v.s. exploitation modalities through fitting experimental data against reinforcement learning models, and (2) to understand the degree in which popular Q-learning-based RL algorithms can truly mimic what we observe within humans.
<div class="row mt-2">
    <div class="col-sm-6 mt-3 mt-md-0">
        {% include figure.html path="assets/img/research/berkeley.png" title="BAIR" class="img-fluid rounded z-depth-1" zoomable=true %}
    </div>
    <div class="col-sm-6 mt-3 mt-md-0">
        <iframe src="/assets/pdf/bair-rl-presentation.pdf" width="100%" height="85%"></iframe>
        <br>
        <p>
            <a href="/assets/pdf/bair-rl-presentation.pdf" target="_blank" rel="noopener noreferrer">
                <!-- You can use an icon or text as a link for the PDF viewer -->
                View Slides (PDF)
            </a>
        </p>
    </div>
</div>

<hr/>

#### **MD Anderson Cancer Center**

Advised by [Dr. Jagan Sastry](https://faculty.mdanderson.org/profiles/jagannadha_sastry.html) and [Dr. Venkatesh Hegde](https://www.researchgate.net/profile/Venkatesh-Hegde). Supported by the [CPRIT CURE](https://www.mdanderson.org/education-training/degrees-programs/summer-research-programs/cprit-cure.html) program. 

**TL;DR:** HPV+ oropharyngeal cancer is rising in prevalence within the U.S. accounting for over 70% of all oropharyngeal cancers. Our group developed an anti-PD1 immunotherapy therapeudic procedure, but we observed that the treatment efficacy differed wildly between tumor implants in the neck v.s. the tongue. My work centered around identifying the differentially expressed genes across these two experiments using RNA-Sequencing Analysis, and eventually constructing a set of hypothesis on the activation/deactivation of biological pathways based on the set of differentially expressed genes. 
<div class="row justify-content-center">
    <div class="col-sm-6">
        {% include figure.html path="assets/img/research/cprit-cure.png" title="CPRIT CURE" class="img-fluid rounded z-depth-1" zoomable=true %}
    </div>
</div>
